server/controllers/finance/reports/debtors/annual_clients_report.js

```diff
@@ -92,6 +111,7 @@ function getDebtorGroupMovements(fiscalYearId, currencyId, rate, hideLockedClien
   const excludeCashClientsCondition = 'AND dg.is_convention = 1 ';
   const sql = `
     SELECT ac.number AS accountNumber, dg.name AS groupName,
+      dg.locked, dg.is_convention AS cashClient,
```

Comment:
This flag feels backwards to me.  If the `is_convention` is ticked, shouldn't that indicate it is not a cash client?  Where does the inversion happen?

EDIT: Nevermind, I see that it is an if/unless statement.  Let's make this either `isConvention` or `IF(dg.is_convention, 0, 1) as cashClient` to clear up the confusion.